docker/config_template.yaml

```diff
@@ -304,6 +304,7 @@ services:
             grpcPort: {{ $temporalGrpcPort }}
             membershipPort: {{ default .Env.FRONTEND_MEMBERSHIP_PORT "6933" }}
             bindOnIP: {{ default .Env.BIND_ON_IP "127.0.0.1" }}
+            httpPort: {{ default .Env.FRONTEND_HTTP_PORT "7243" }}
```

Comment:
You might want to check if there are any changes to be made here:
https://github.com/search?q=repo%3Atemporalio%2Fdocker-compose%207233&type=code
or here:
https://github.com/search?q=repo%3Atemporalio%2Fhelm-charts%207233&type=code
to expose the port

Reply:
:+1: Will open PRs to those upon merge here.